site/ja/xla/overview.md

```diff
@@ -0,0 +1,56 @@
+# XLAとは？
+
+<div style="width:50%; margin:auto; margin-bottom:10px; margin-top:20px;">
+<img style="width:50%" src="https://raw.githubusercontent.com/tensorflow/tensorflow/master/tensorflow/compiler/xla/g3doc/images/xlalogo.png">
+</div>
+
+> 注意：XLAは現在開発中であるため、特定の状況でメモリ使用量の増大や性能の悪化を引き起こす場合があります。
+
+XLAは線形代数の演算に特化したコンパイラで、XLAを使うことでTensorFlowの演算を最適化し、メモリ使用量の削減や性能の向上を期待できます。  
+XLAは、実行時にTensorFlowの計算グラフをコンパイルして実行する [just-in-time (JIT) コンパイル機能](https://www.tensorflow.org/xla/jit) と、TensorFlowの計算グラフを実行コードにコンパイルする [ahead-of-time (AOT) コンパイル機能](https://www.tensorflow.org/xla/tfcompile) の2つの機能を提供しています。  

```

Comment:
Last paragraph and some sentences which mention that XLA is experimental are missing.

@lamberta Is there a some rules or standards for, how faithful the translation should be to the original one? Since translation by community, I think we should balance the faithfulness and cost.

Reply:
Good question.
I think mentioning experimental is important here since that seems like something a Japanese dev would want to know before investing time.

But we should perhaps come up with some rules and standards for what gets translated and what is considered out-of-date.
The community is welcome to add translation tips in the [ja/README.md](https://github.com/tensorflow/docs/tree/master/site/ja) (you can see his is what the Korean translation folks are doing: [ko/README.md](https://github.com/tensorflow/docs/tree/master/site/ko))

Suggestions?

Reply:
Thanks for your comment.
I agree with including the sentence which mention that XLA is experimental.

For now, I translate the document to match original document as possible, because there is no documentation guideline.

Reply:
Thank you. Our current guidelines are here: https://www.tensorflow.org/community/contribute/docs#community_translations
> The English docs are the source-of-truth and translations should follow these guides as close as possible. That said, translations are written for the communities they serve. If the English terminology, phrasing, style, or tone does not translate to another language, please use a translation appropriate for the reader.

Technical content should remain as close as possible but style is not so rigid. I'd prefer it read naturally.